src/termio/stream_handler.zig

```diff
@@ -53,6 +53,9 @@ pub const StreamHandler = struct {
     /// The color reporting format for OSC requests.
     osc_color_report_format: configpkg.Config.OSCColorReportFormat,
 
+    /// The config conditional state.
+    config_conditional_state: configpkg.ConditionalState,
```

Comment:
I'm unsure if this is the desired outcome of if the underlying scheme value should be pulled in directly. Let me know if this should be changed.